Automattic/jetpack
# Subscription Block: Add success message

Fixes #21187

#### Changes proposed in this Pull Request:

* Updated `projects/plugins/jetpack/extensions/blocks/subscriptions` to support teaxtarea for `success_message` in the widget setttings.
* Updated `projects/plugins/jetpack/modules/subscriptions/views.php` to support `success_message` when passed as an attribute to the shortcode.
* Updated test cases of subscription form extension

#### Jetpack product discussion NA



#### Does this pull request to change what data or activity we track or use? NA



#### Testing instructions:






* Checkout to the PR branch
* Build subscription extension
* Enable jetpack subscription under discussion tab of jetpack settings
* Go to appearance -> customize -> widgets -> footer#1 -> add a legacy widget subscription block (if it not already transformed to subscription block and then transform it to subscription block)
* Select subscription block and click on show more settings and scroll down
* Edit success message and publish it.
* Open the site in incognito mode and then enter an email address
* Verify success message.

#### Screenrecording https://youtu.be/Z5cSgh5neQA
